docs/quality-assurance-test-cases.md

```diff
@@ -173,7 +185,16 @@ This document contains some examples of test cases for each feature implemented
 | Logged in user tries to complete a task from an non existing request (The request  exists in a different relationship) | Fail |
 | Logged in user tries to complete a task from a request which is not in the ACCEPTED state (as a mentor or as a mentee) | Fail |
 
+### Delete Task
 
+**Service:** DELETE /mentorship_relation/{request_id}/task/{task_id}
+| Test Case       | Outcome |
```

Comment:
A nitpick. Can you uniformly space out the table in markdown for uniformity with other tables?

Reply:
When I check on github, it looks fine. Do you mean changing the spacing at the code (as shown below)?
![DeleteTask](https://user-images.githubusercontent.com/31242926/93352607-ec9d3380-f832-11ea-8fac-0b641ca0946e.png)

Reply:
Yes, precisely.